net/wireguard/src/opnsense/mvc/app/models/OPNsense/Wireguard/Server.xml

```diff
@@ -31,6 +31,9 @@
                     <default>51820</default>
                     <Required>Y</Required>
                 </port>
+                <dns type="CSVListField">
```

Comment:
No validation?

Reply:
Hm, wasn't this the thing with v4 and v6 that doesn't Work?

Reply:
Took the one from Zabbix Agent and removed the hostname stuff.

Reply:
Ok, thanks! Please talk to @AdSchellevis about finishing the built-in validation or how to use it.